packages/gestalt/src/contexts/Theme.js

```diff
@@ -0,0 +1,132 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+export type ColorScheme = 'light' | 'dark' | 'userPreference';
+
+export const ColorSchemePropType: React$PropType$Primitive<ColorScheme> = PropTypes.oneOf(
+  ['light', 'dark', 'userPreference']
```

Comment:
I wonder if `userPreference` here is better expressed as the null value? If not, this needs an `isRequired`.

Reply:
The `colorScheme` prop is optional. If no scheme is specified it defaults to `light` for now since dark mode looks pretty bad in some places. Once we have dark mode stable `userPreference` would be a better default